nmdc_server/migrations/versions/e54d37bfb90b_rename_data_object_workflow_types.py

```diff
@@ -0,0 +1,78 @@
+"""Rename data object workflow types
+
+The Berkeley schema migration changed a number of workflow type names. Even though
+the `data_object` table is truncated at the start of the ingest process, the old
+workflow types were being introduced by a merge operation from the production database
```

Comment:
I don't know what the correct term here is. I find "production" to be ambiguous because both databases would be in the "production" environment. I put "live" here as a suggestion.

```suggestion
workflow types were being introduced by a merge operation from the live database
```